libraries/core_libs/consensus/src/dag/sortition_params_manager.cpp

```diff
@@ -116,7 +116,7 @@ void SortitionParamsManager::pbftBlockPushed(const SyncBlock& block, DbStorage::
     if (non_empty_pbft_chain_size % config_.changing_interval == 0) {
       const auto params_change = calculateChange(period);
       params_changes_.push_back(params_change);
-      db_->saveSortitionParamsChange(period, std::move(params_change), batch);
+      db_->saveSortitionParamsChange(period, params_change, batch);
```

Comment:
I think it would be better to change it so we can use move here. If I have check where it is used, it should not be a problem

Reply:
- in this concrete line we would not benefit from move as saveSortitionParamsChange just takes the rlp so if you take it from rvale&& or const lvalue& does not make much difference
- + SortitionParamsChange class is primitive type, which means it does not contain any data that would use pointer so moing this object always does a copy anyway